src/common/hooks/useCareApps.ts

```
@@ -22,6 +22,14 @@ export const useCareAppNavItems = () => {
   return navItems;
 };
 
+export const useCareAppConsultationTabs = () => {
+  const careApps = useCareApps();
+
+  return careApps.reduce((acc, app) => {
+    return { ...acc, ...(app.consultationTabs ?? {}) };
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)